tests/test_maxcut.py

```diff
@@ -54,7 +54,26 @@ def test_maxcut_qaoa_obj_fixed_angles():
             gamma, beta, AR = get_fixed_gamma_beta(d, p, return_AR=True)
             for simulator in ["auto", "qiskit"]:
                 f = get_qaoa_maxcut_objective(N, p, G=G, parameterization="gamma beta", simulator=simulator)
-                assert -f(gamma, beta) / optimal_cut > AR
+                assert f(gamma, beta) / optimal_cut < AR
```

Comment:
This should be `f(gamma, beta) / optimal_cut > AR` if `f(gamma, beta)` is positive.